back-end/server/helpers.js

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
it very strange that the library to upload images doesn't check that :)

Reply:
lol yeah, i think the library is for uploading any kind of file! it may have some built-in methods for determining file type, but i didn't find em :P i'm not even sure what would happen if you tried to upload a non-image file right now!